tagsets/experiments.php

```diff
@@ -336,7 +336,7 @@ function experiment__experiments_format_alist($alist) {
         <TD class="small" align="center">';
             if ($experiment_type=="laboratory") {
                 echo lang('sessions').': <span style="font-weight: bold;" title="number of sessions">'.$num_sessions.'</span>';
-                echo '&nbsp&nbsp&nbsp<span title="number of subjects assigned / enroled / participated"><i class="fa fa-users" style="font-size: 7pt; padding-right: 2px;"></i>'.
+                echo '&nbsp&nbsp&nbsp<span title="number of subjects assigned / enrolled / participated"><i class="fa fa-users" style="font-size: 7pt; padding-right: 2px;"></i>'.
```

Comment:
Please remove this from the commit. It has nothing to do with the issue, and it's a US vs. UK English thing.